### DISCUSSION TITLE
我落入清凉的酷暑中

### DISCUSSION THREAD

**Discussion by giscus** [Thrimbda/blog-comment#2] [General]
# 我落入清凉的酷暑中

c1 的博客

http://127.0.0.1:1111/blog/wo-luo-ru-qing-liang-de-ku-shu-zhong/

**Comment by Thrimbda** [comment]
我试试评论！

**Reply by Thrimbda** [reply]
tnnd 我调试的时候发了评论给我干成 127.0.0.1 了啊啊啊啊啊啊啊啊啊啊我不活了
